db/migrate/20190315055139_create_volunteers.rb

```diff
@@ -14,7 +14,20 @@ def change
       t.string :state
       t.string :employer
       t.string :position
-      t.string :availability
+      t.string :avail_sun_am
+      t.string :avail_sun_pm
+      t.string :avail_mon_am
+      t.string :avail_mon_pm
+      t.string :avail_tue_am
+      t.string :avail_tue_pm
+      t.string :avail_wed_am
+      t.string :avail_wed_pm
+      t.string :avail_thr_am
+      t.string :avail_thr_pm
+      t.string :avail_fri_am
+      t.string :avail_fri_pm
+      t.string :avail_sat_am
+      t.string :avail_sat_pm
```

Comment:
Please make these changes to the database by creating a new migration instead of editing an existing one.